app/keyWatcher/index.js

```diff
@@ -1,35 +1,34 @@
-const { createNodeApi } = require('./api')
-const { setupKeyWatcher } = require('./keyWatcher')
-const { ConnectionError } = require('../utils/Errors')
+import { setupKeyWatcher as setup } from './keyWatcher.js'
+import { ConnectionError } from '../utils/Errors.js'
 
-module.exports = {
```

Comment:
This is fine, my general preference is `export const` declarations anyway, but remember you can do `export { item1, item2 }` which here would have produced less churn. Just an FYI

Reply:
thanks, didn't realise!